build.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
Ideally, these should be put into `build-commands` instead of being separate script.

Reply:
Okay, will do.